### ISSUE TITLE
Add an option to the io_loop fixture to configure ioloop maker

### ISSUE THREAD

**Issue by eugeniy** [eugeniy/pytest-tornado#8]
Needed for apps that rely on IOLoop.instance()

Currently, IOLoop() is created, this should be overridable.

**Comment by holljen** [comment]
Yes please, I'd like this. I'm using zmq in addition to tornado and it relies on IOLoop.instance(). I've solved it by calling io_loop.install() but it isn't very pretty :)
